### PR TITLE
fix: Enhance and place graphic onto about page

### LINKED ISSUE #4404: [Site] Enhance and place the graphic onto the about page.
### Current Behavior
The About page - http://layer5.io/about doesn't reflect the stats of the community and project.

### Desired Behavior
Enhance this existing graphic from this page - https://layer5.io/blog/community/meshery-surpasses-1000-contributors representing cumulative totals from all of Layer5's projects not just Meshery alone onto the About page.
Reference Slack thread - https://layer5io.slack.com/archives/C015QJKUMPU/p1687195434476819

Note - The issue description will be updated with new statistics to add to the graphic.

### Screenshots / Mockups
Enhance this graphic representing the cumulative totals from all of Layer5's projects too, not just Meshery.
Figma link to design - https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?type=design&node-id=7999-21344&t=7h4iYOk56wjeS06v-0

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer). his blog post’s graphic can be placed onto the <http://layer5.io/about|layer5.io/about> page - <https://layer5.io/blog/community/meshery-surpasses-1000-contributors>

[Slack Message](https://layer5io.slack.com/archives/C015QJKUMPU/p1687195434476819?thread_ts=1687195434.476819&cid=C015QJKUMPU)

### PR DESCRIPTION
**Pull request by anikdhabal**
**Description**

This PR fixes #4404 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8592d9fdc14827cb4a737cf6c61a39215ee2f7fb at: https://64a117f10e35132b944027e5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e2b29e38bf3fc977e35416448176fedf51fb7f04 at: https://64a16494bad6fb43ba41281f--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@anikdhabal Please sign off your commits. 
Follow the instructions: https://github.com/layer5io/layer5/pull/4471/checks?check_run_id=14717816428

**Comment by anikdhabal** [comment]
> @anikdhabal Please sign off your commits. Follow the instructions: https://github.com/layer5io/layer5/pull/4471/checks?check_run_id=14717816428
> 
> Also, let's discuss this on websites meeting today at 5:30 PM IST. Add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

Yes, I've already sign of in my latest commit.

**Comment by samyakshah3008** [comment]
@anikdhabal  Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

**Comment by anikdhabal** [comment]
@leecalcote can you review this PR?

**Comment by l5io** [comment]
🚀 Preview for commit f4934d55628fa2bafe429262a3a8f2125ee78bfa at: https://64aeea51b7f58010877fad13--layer5.netlify.app

File: src/sections/Company/About/images/layer5-community-stat.svg
Comment (leecalcote) [inline src/sections/Company/About/images/layer5-community-stat.svg:1]: Thanks for this effort.

We can't have an outline around the our mascot (the stick figure).

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: src/sections/Company/About/images/layer5-community-stat.svg
Comment (anikdhabal) [reply src/sections/Company/About/images/layer5-community-stat.svg:1]: ok, thank you

**Comment by l5io** [comment]
🚀 Preview for commit 2cf04f17af2c5b5a8eaf72b060530558df15eb7d at: https://64b22379c648544b91359742--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit dd17b8a7dcaa1d1e6f356e24b6b7e2e265206054 at: https://64b826a608cd9e0c2bb953f7--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5079fdb906760d48a426ddecdc6c6fb87b99762a at: https://64b82bc57d7c3312f9c631b2--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]
Excellent, @anikdhabal

**Comment by leecalcote** [comment]
@anikdhabal, let's do this: replace the CNCF logo with this new infographic. The infographic should be displayed smaller than it currently is.

**Comment by anikdhabal** [comment]
> @anikdhabal, let's do this: replace the CNCF logo with this new infographic. The infographic should be displayed smaller than it currently is.

Done

**Review by samyakshah3008 — APPROVED** [review]

**Comment by Chadha93** [comment]
Thank you gents.

**Pull request merged**